webapp/src/ts/modules/home/home.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AuthService } from '@mm-services/auth.service';
+import { HeaderTabsService } from '@mm-services/header-tabs.service';
+
+@Component({
+  template: '',
+})
+export class HomeComponent implements OnInit {
```

Comment:
It'd be nice to unit test this. Could you please migrate the unit test too? :)
https://github.com/medic/cht-core/blob/master/webapp/tests/karma/unit/controllers/home.js

Reply:
ups sorry, I didn't see it :+1:

Reply:
Awesome thank you!